composer.json

```diff
@@ -8,8 +8,8 @@
   },
   "require-dev": {
     "phpunit/phpunit": "^6",
-    "plentymarkets/plugin-ceres": "5.0.0",
-    "plentymarkets/plugin-io": "5.0.0",
+    "plentymarkets/plugin-ceres": "5.0.*",
```

Comment:
Please have a look at [this branch](https://github.com/findologic/plugin-plentymarkets-ceres-api/tree/improve_versions). You may want to cherry-pick the commits in there and add them to here. It basically should fix the build regarding the version issues.